### PR TITLE
Ability to zoom / open images fullscreen #23

### PR DESCRIPTION
**Pull request by YorgunKelebek**
Addressing issue #23 Ability to zoom / open images fullscreen

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -101,14 +101,30 @@ function buildBlogSummary(blog, modularContent)
...
+    body = convertImagesToHyperlink(body);
 	blogSummary.querySelector(".summary-body").innerHTML = body;
 
     processContentSnippets(blogSummary, modularContent);
 	return blogSummary;
 }
 
 
+function convertImagesToHyperlink(blogBody) {
+    let wrapper = document.createElement("div");
```
Comment (goofballLogic) [inline js/app.js]: this could be const (other instances below).

File: js/app.js [hunk 2]
```diff
@@ -101,14 +101,30 @@ function buildBlogSummary(blog, modularContent)
...
+    wrapper.innerHTML = blogBody;
+    wrapper.querySelectorAll("img").forEach(image => {
+        let elementHyperlink = document.createElement('a');
```
Comment (goofballLogic) [inline js/app.js]: you could use a template for this i suppose fwiw

File: js/app.js (hunk 1 above)
Comment (YorgunKelebek) [reply js/app.js]: Yes, you're right

File: js/app.js (hunk 2 above)
Comment (YorgunKelebek) [reply js/app.js]: It may keep it consistent... I'll think about it, for another time

**Pull request merged**